api/src/main/java/io/opentelemetry/common/Attributes.java

```diff
@@ -231,105 +231,105 @@ public Attributes build() {
+    public Builder set(String key, boolean value) {
+      return set(booleanKey(key), value);
     }
 
     /**
      * Sets a String array attribute into this.
      *
-     * <p>Note: It is strongly recommended to use {@link #setAttribute(AttributeKey, Object)}, and
+     * <p>Note: It is strongly recommended to use {@link #set(AttributeKey, Object)}, and
      * pre-allocate your keys, if possible.
      *
      * @return this Builder
      */
-    public Builder setAttribute(String key, String... value) {
-      return setAttribute(stringArrayKey(key), value == null ? null : Arrays.asList(value));
+    public Builder set(String key, String... value) {
+      return set(stringArrayKey(key), value == null ? null : Arrays.asList(value));
     }
 
     /**
      * Sets a Long array attribute into this.
      *
-     * <p>Note: It is strongly recommended to use {@link #setAttribute(AttributeKey, Object)}, and
+     * <p>Note: It is strongly recommended to use {@link #set(AttributeKey, Object)}, and
      * pre-allocate your keys, if possible.
      *
      * @return this Builder
      */
-    public Builder setAttribute(String key, Long... value) {
-      return setAttribute(longArrayKey(key), value == null ? null : Arrays.asList(value));
+    public Builder set(String key, Long... value) {
+      return set(longArrayKey(key), value == null ? null : Arrays.asList(value));
     }
 
     /**
      * Sets a Double array attribute into this.
      *
-     * <p>Note: It is strongly recommended to use {@link #setAttribute(AttributeKey, Object)}, and
+     * <p>Note: It is strongly recommended to use {@link #set(AttributeKey, Object)}, and
      * pre-allocate your keys, if possible.
      *
      * @return this Builder
      */
-    public Builder setAttribute(String key, Double... value) {
-      return setAttribute(doubleArrayKey(key), value == null ? null : Arrays.asList(value));
+    public Builder set(String key, Double... value) {
+      return set(doubleArrayKey(key), value == null ? null : Arrays.asList(value));
     }
 
     /**
      * Sets a Boolean array attribute into this.
      *
-     * <p>Note: It is strongly recommended to use {@link #setAttribute(AttributeKey, Object)}, and
+     * <p>Note: It is strongly recommended to use {@link #set(AttributeKey, Object)}, and
      * pre-allocate your keys, if possible.
      *
      * @return this Builder
      */
-    public Builder setAttribute(String key, Boolean... value) {
-      return setAttribute(booleanArrayKey(key), value == null ? null : Arrays.asList(value));
+    public Builder set(String key, Boolean... value) {
+      return set(booleanArrayKey(key), value == null ? null : Arrays.asList(value));
     }
 
     /**
-     * Add all the provided attributes to this Builder.
+     * Sets all the provided attributes into this Builder.
      *
      * @return this Builder
      */
-    public Builder addAll(Attributes attributes) {
+    public Builder setAll(Attributes attributes) {
```

Comment:
@jkwatson Any reason to use `addAll` before? Couldn't think of one so made this symmetrical with above

Reply:
But this does not `set` as in "replace current value", it actually `adds` given attributes to already existing ones. No?

Reply:
last-in wins, so this will replace current values. `setAll` is a better name. 👍